result/isislsp.yaml

```diff
@@ -0,0 +1,380 @@
+        extended_ipv4_reachabilities:
+          description: |-
+            This container defines list of IPv4 extended reachability information.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.ExtendedIpv4Reachability'
+        ipv6_reachabilities:
+          description: |-
+            This container defines list of IPv6 reachability information.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.Ipv6Reachability'
+
+    IsisLsp.Flags:
+      description: >-
+        LSP Type flags.
+      type: object
+      properties:
+        partition_repair:
+          description: |-
+            When set, the originator supports partition repair.
+          type: boolean
+        attached_error:
+          description: |-
+            When set, the originator is attached to another area using the referred metric.
+          type: boolean
+        attached_expense:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_delay:
+          description: |-
+            Delay Metric - when set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_default:
+          description: |-
+            Default Metric - when set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        overload:
+          description: |-
+            Overload bit - when set, the originator is overloaded, and must
+            be avoided in path calculation.
+          type: boolean
+
+    IsisLsp.IsReachability:
+      description: >-
+        This container describes list of ISIS neighbors and attributes in IS-Reachability TLV 2.
+      type: object
+      properties:
+        neighbors: 
+          description: >-
+            This container describes Intermediate System (IS) neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.neighbor'
+
+    IsisLsp.ExtendedIsReachability:
+      description: >-
+        This is list of ISIS neighbors and attributes in Extended-IS-Reachability TLV 22.
+      type: object
+      properties:
+        neighbors: 
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.neighbor'
+
+    IsisLsp.neighbor:
+      description: >-
+        This contains IS neighbors.
+      type: object
+      properties:
+        system_id:
+          description: >-
+            The System ID for this emulated ISIS router, e.g. "640100010000".
+          type: string
+          format: hex
+
+    IsisLsp.Ipv4InternalReachability:
+      description: |-
+        This container defines list of IPv4 internal reachability information.
```

Comment:
This container defines the information present on one IPv4 Internal Reachability TLV.